scala/scala_import.bzl

```diff
@@ -109,10 +116,17 @@ scala_import = rule(
     implementation = _scala_import_impl,
     attrs = {
         "jars": attr.label_list(
-            allow_files = True
+            allow_files = True,
         ),  #current hidden assumption is that these point to full, not ijar'd jars
         "deps": attr.label_list(),
         "runtime_deps": attr.label_list(),
-        "exports": attr.label_list()
+        "exports": attr.label_list(),
+        "neverlink": attr.int(
```

Comment:
I kind of wish we could have bool here. I guess consistency with java rules is more important but I assume their parser didn’t support bool at that time.